packages/edit-site/src/components/layout/style.scss

```diff
@@ -300,3 +293,16 @@
 	}
 
 }
+
+.edit-site-layout__area {
```

Comment:
This is the style that make the area look like a rounded frame. This style maybe be shared with the "preview" as well.